resources/public/api/conf/1.0/testOnly/application.yaml

```
@@ -1141,8 +1141,7 @@ paths:
                   $ref: "#/components/examples/errors.Submission.GenericServerError"
                 No Subscription Data:
                   $ref: "#/components/examples/errors.Submission.NoSubscriptionData"
-      description: |
-        Submit an Overseas Return Notification
+      description: "Submit an Overseas Return Notification\n### Error Codes\n<p>The errors listed here return a 422 HTTP code when the Submit Overseas Rreturn Notification request is not processed successfully.</p> \n<table>\n<thead>\n<tr>\n<th>Error Message</th>\n<th>Next Steps</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>003 - Request could not be processed</td>\n<td>The message could not be processed due to locking. Please check if there is currently an open enquiry on your tax account, and that the <em>filedDateGIR</em> date is not in the future.</td>\n</tr>\n<tr>\n<td>044 - Tax Obligation already fulfilled</td>\n<td>A BTN for the specified period has already been submitted.</td>\n</tr>\n<tr>\n<td>063 - Business Partner does not have an Active Pillar 2 Subscription</td>\n<td>The Pillar 2 ID in the request header is not listed as active by HMRC for this regime. Please contact the central support team.</td>\n</tr>\n<tr>\n<td>093 - Invalid Return</td>\n<td>There is an error with the return you’ve submitted in the request.</td>\n</tr>\n</tbody>\n</table>\n"
```

```suggestion
      description: "Submit an Overseas Return Notification\n### Error Codes\n<p>The errors listed here return a 422 HTTP code when the Submit Overseas Return Notification request is not processed successfully.</p> \n<table>\n<thead>\n<tr>\n<th>Error Message</th>\n<th>Next Steps</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td>003 - Request could not be processed</td>\n<td>The message could not be processed due to locking. Please check if there is currently an open enquiry on your tax account, and that the <em>filedDateGIR</em> date is not in the future.</td>\n</tr>\n<tr>\n<td>044 - Tax Obligation already fulfilled</td>\n<td>A BTN for the specified period has already been submitted.</td>\n</tr>\n<tr>\n<td>063 - Business Partner does not have an Active Pillar 2 Subscription</td>\n<td>The Pillar 2 ID in the request header is not listed as active by HMRC for this regime. Please contact the central support team.</td>\n</tr>\n<tr>\n<td>093 - Invalid Return</td>\n<td>There is an error with the return you’ve submitted in the request.</td>\n</tr>\n</tbody>\n</table>\n"
```

Minor type `Rreturn`

---

generating the spec should resolve that 👍🏻